### PR TITLE
Feat/repository flow

### PR DESCRIPTION
**Pull request by sudoFerraz**
## Description
This PR adds the repository flow

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app/api/github/repo/registered/fetchRegisteredRepos.ts [hunk 1]
```diff
@@ -11,11 +11,13 @@ export async function fetchRegisteredGitRepos(
   teamId: string,
 ): Promise<RegisteredGitRepo[]> {
   try {
+    console.log("fetching registered repos for team", teamId);
```
Comment (AlecErasmus) [inline src/app/api/github/repo/registered/fetchRegisteredRepos.ts]: Left console log

File: src/app/api/github/repo/registered/fetchRegisteredRepos.ts [hunk 2]
```diff
@@ -11,11 +11,13 @@ export async function fetchRegisteredGitRepos(
...
     const foundRegisteredRepos = await prisma.githubRepo.findMany({
       where: {
         team_id: teamId,
       },
     });
+    console.log("found registered repos", foundRegisteredRepos);
```
Comment (AlecErasmus) [inline src/app/api/github/repo/registered/fetchRegisteredRepos.ts]: Left console log

File: src/components/navigation/sidebar.tsx [hunk 3]
```diff
@@ -12,8 +12,8 @@ export default function Sidebar() {
       <div className="space-y-4 py-4">
         <div className="px-3 py-2">
           <div className="space-y-1">
-            <h2 className="mb-2 px-4 text-xl font-semibold tracking-tight">
-              Overview
+            <h2 className="mb-2 px-4 text-2xl font-semibold tracking-tight pb-4">
+              Armitage
```
Comment (AlecErasmus) [inline src/components/navigation/sidebar.tsx:16]: This is correct?

File: src/components/overview/dashboardInsights.tsx [hunk 4]
```diff
@@ -19,8 +19,8 @@ export function DashboardInsights({ overview }: DashboardInsightsProps) {
             {overview?.totalScore.toFixed(2)}
           </div>
           <p className="pt-1 text-xs text-muted-foreground">
-            {overview?.weekGrowth ? (overview?.weekGrowth > 0 ? "+" : "-") : ""}{" "}
-            {overview?.weekGrowth.toFixed(2)}% all time growth during last week
+            {overview?.weekGrowth?.toFixed(2)}% all time growth during current
```
Comment (AlecErasmus) [inline src/components/overview/dashboardInsights.tsx]: You will need to update,  this works

overview?.weekGrowth?.toFixed(2) | 0

File: src/constants/data.ts [hunk 5]
```diff
@@ -5,19 +5,31 @@ export const signedInNavItems: NavItem[] = [
...
-    icon: "kanban",
-    label: "employee",
+    icon: "heartHandshake",
+    label: "contributors",
+  },
+  {
+    title: "Feedback",
+    href: "https://armitage.canny.io/feedback",
+    icon: "help",
+    label: "help",
```
Comment (AlecErasmus) [inline src/constants/data.ts]: This has been removed, think this is a rebase. mistake

File: src/components/navigation/sidebar.tsx (hunk 3 above)
Comment (sudoFerraz) [reply src/components/navigation/sidebar.tsx:16]: yeah, this is the title of the menu bar, made no sense before that it was "Overview" on the menu, it should be our name 😅 
image

**Review by AlecErasmus — APPROVED** [review]

**Pull request merged**